debezium-core/src/main/java/io/debezium/pipeline/spi/ChangeRecordEmitter.java

```diff
@@ -40,6 +40,10 @@ public interface ChangeRecordEmitter<P extends Partition> {
      */
     Operation getOperation();
 
+    default boolean shouldIgnoreRecords() {
```

Comment:
I'd suggest to use `shouldIgnoreRecord()` (singular `Record`) or just `ignoreRecord()`.
Also please add javadoc to this method.

Reply:
According to class doc https://github.com/debezium/debezium/blob/main/debezium-core/src/main/java/io/debezium/pipeline/spi/ChangeRecordEmitter.java#L15, it emits one or more records and mostly one record.
Let me know if i can go ahead update the method name to `ignoreRecord()`